one_fm/api/api.py

```diff
@@ -118,15 +121,18 @@ def store_fcm_token(employee_id ,fcm_token,device_os):
     except Exception as e:
         print(frappe.get_traceback())
 
-"""
+
+@frappe.whitelist()
+def push_notification(employee_id, title, body, checkin, arriveLate ,checkout):

```

Comment:
This function is specific to check-in/arrive late. Follow appropriate naming for the method.